test_single_controller_ppo.py

```diff
@@ -222,7 +222,6 @@ def build_train_config(self, pretrain_model_name: str):
             'max_seq_len': self.max_seq_len,
             'python_log_level': 'debug',
             'console_log_interval': '1ba',
-            'eval_interval': '1iter',
```

Comment:
TODO: Need to handle this inside `_run_evaluation` but want to wait on the yaml code first to make things easier